addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/service/dhcp/ReceiveDHCPRequestPackets.java

```diff
@@ -109,17 +110,17 @@ public void run() {
                 String requestedAddressStr = requestedAddress.getHostAddress();
                 StateUpdate receiver = registeredListeners.get(requestedAddressStr);
                 if (receiver != null) {
-                    logger.info("DHCP request for registered address: " + requestedAddressStr);
+                    logger.info("DHCP request for registered address: {}", requestedAddressStr);
                     receiver.newState(0);
                 } else {
-                    logger.info("DHCP request for unknown address: " + requestedAddressStr);
+                    logger.info("DHCP request for unknown address: {}", requestedAddressStr);
                 }
             }
         } catch (IOException e) {
             if (willbeclosed) {
                 return;
             }
-            logger.error(e.getLocalizedMessage());
+            logger.error("{}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace